java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -37,6 +38,25 @@ public interface StringBaseCommands {
      */
     CompletableFuture<String> get(String key);
```

Comment:
I think you can remove this get (maybe in the next PR)

Reply:
You need to update benchmarking app when you do that.

Reply:
I am thinking of keeping this for convenience purposes (it is easy for end user to use directly strings). But we can decide on this later